app/(auth)/PageTitleProvider.tsx

```diff
@@ -0,0 +1,23 @@
+// app/PageTitleContext.tsx
+import React, { createContext, useContext, useState } from 'react';
+
+
+const PageTitleContext = createContext<{ setTitle: (title: string) => void, title: string } | undefined>(undefined);
+
+export const PageTitleProvider: React.FC<{ children: React.ReactNode }> = ({ children }) => {
+    const [title, setTitle] = useState<string>('服事表');
```

Comment:
just a simple example, a little ugly.
```suggestion
    const pathName = usePathName();
    const titleMap = { '/full-calendar': '服事總表', '/dashboard': '個人服事表' }  
    const title = titleMap[pathName]

    const [title, setTitle] = useState<string>(title);
```